src/sink/wavefront.rs

```diff
@@ -215,6 +308,7 @@ impl Sink for Wavefront {
     }
 
     fn flush(&mut self) {
+        self.format_stats(time::now());
```

Comment:
In this sink we don't actually use the passed in time. I'll remove it altogether.